packages/react-core/src/components/Pagination/OptionsToggle.tsx

```diff
@@ -37,12 +37,14 @@ export interface OptionsToggleProps extends React.HTMLProps<HTMLDivElement> {
   onEnter?: () => void;
   /** Label for the English word "of" */
   ofWord?: string;
+  /** Component to be used for wrapping the toggle contents. */
+  component?: 'div' | 'button';
```

Comment:
I think we should specify here that `button` should be used to make entire toggle text clickable.